src/modules/fancyzones/tests/UnitTests/JsonHelpers.Tests.cpp

```diff
@@ -1011,7 +1017,7 @@ namespace FancyZonesUnitTests
                 for (int i = 0; i < 10; i++)
                 {
                     json::JsonObject obj = json::JsonObject::Parse(m_defaultCustomDeviceStr);
-                    obj.SetNamedValue(L"device-id", json::JsonValue::CreateStringValue(std::to_wstring(i) + L"_1920_1200_{39B25DD2-130D-4B5D-8851-4791D66B1539}"));
+                    obj.SetNamedValue(L"device-id", json::JsonValue::CreateStringValue(std::to_wstring(i) + L"#_1920_1200_{39B25DD2-130D-4B5D-8851-4791D66B1539}"));
```

Comment:
Please see https://github.com/microsoft/PowerToys/issues/1827
We can't force the `#` to be present since it is currently supported and several users may have it without it.

Reply:
Is a device id *without #* actually a real-case scenario? I am happy to search for #, and only then use that logic.

Reply:
Yes, it's a real case scenario as you can see in this issue https://github.com/microsoft/PowerToys/issues/1777#issuecomment-606538165, where Windows fails to return a proper ID for a connected monitor.

Reply:
Pushed a fix.

Reply:
Excellent!